sdk-manifests/ably-dotnet.yaml

```diff
@@ -0,0 +1,110 @@
+%YAML 1.2
+---
+common-version: 1.2.0-alpha.1
+compliance:
+  Authentication:
+    API Key:
+    Token:
+      Callback:
+      Literal:
+      URL:
+        Query Time:
+  Debugging:
+    Error Information:
+    Logs:
+  Protocol:
+    JSON:
+  Realtime:
+    Authentication:
+      Get Confirmed Client Identifier:
+    Channel:
+      Attach:
+      Encryption:
+      History:
+      Mode:
+      Presence:
+        Enter:
+          Client:
+        Get:
+        History:
+        Subscribe:
+        Update:
+          Client:
+      Publish:
+      Retry Timeout:
+      State Events:
+      Subscribe:
+        Deltas:
+        Rewind:
+    Connection:
+      Disconnected Retry Timeout:
+      Get Identifier:
+      Lifecycle Control:
+      Ping:
+      Recovery:
+      State Events:
+      Suspended Retry Timeout:
+    Message Echoes:
+    Message Queuing:
+    Push Notifications:
+      Local Device State:
+    Transport Parameters:
+  REST:
+    Authentication:
+      Authorize:
+      Create Token Request:
+      Get Client Identifier:
+      Request Token:
+    Channel:
+      Encryption:
+      Existence Check:
+      Get:
+      History:
+      Iterate:
+      Name:
+      Presence:
+        History:
+        Member List:
+      Publish:
+        Idempotence:
+      Push Notifications:
+        List Subscriptions:
+        Subscribe:
+      Release:
+      Status:
+        Channel Details:
+        Enumerate:
```

Comment:
I don't think this feature is implemented in ably-dotnet (or any SDK). Just to be clear this one is about using the channel enumeration API - not about enumerating local RestChannel instances.

Reply:
Actually, I had a doubt about this one, I will provide link to the `enumerator` 
https://github.com/ably/ably-dotnet/blob/b6088aa985068535ce6f30f072399aea612fc478/src/IO.Ably.Shared/Realtime/RealtimeChannels.cs#L193-L206

Reply:
Okay, seems it's a part of `Status->Enumerate` API, I will update

Reply:
Removed